docs/foundations/color/accessibility.md

```diff
@@ -20,61 +33,145 @@ accessible to everyone.
+        <td data-label="Large text"><span class="pass">WCAG AA: Pass</span></td>
+        <td data-label="Objects and UI components"><span class="pass">WCAG AA: Pass</span></td>
+      </tr>
+      <tr>
+        <td data-label="Background color"><code>gray-70</code> (<code>#383838</code>)</td>
+        <td data-label="Contrast ratio"><span class="fail">2.58</span></td>
+        <td data-label="Normal text"><span class="fail">WCAG AA: Fail</span></td>
+        <td data-label="Large text"><span class="fail">WCAG AA: Fail</span></td>
+        <td data-label="Objects and UI components"><span class="fail">WCAG AA: Fail</span></td>
+      </tr>
+      <tr>
+        <td data-label="Background color"><code>gray-80</code> (<code>#292929</code>)</td>
+        <td data-label="Contrast ratio"><span class="fail">3.21</span></td>
+        <td data-label="Normal text"><span class="fail">WCAG AA: Fail</span></td>
+        <td data-label="Large text"><span class="pass">WCAG AA: Pass</span></td>
+        <td data-label="Objects and UI components"><span class="pass">WCAG AA: Pass</span></td>
+      </tr>
+      <tr>
+        <td data-label="Background color"><code>gray-90</code> (<code>#1f1f1f</code>)</td>
+        <td data-label="Contrast ratio"><span class="fail">3.63</span></td>
+        <td data-label="Normal text"><span class="fail">WCAG AA: Fail</span></td>
+        <td data-label="Large text"><span class="pass">WCAG AA: Pass</span></td>
+        <td data-label="Objects and UI components"><span class="pass">WCAG AA: Pass</span></td>
+      </tr>
+      <tr>
+        <td data-label="Background color"><code>gray-95</code> (<code>#151515</code>)</td>
+        <td data-label="Contrast ratio"><span class="fail">4.03</span></td>
+        <td data-label="Normal text"><span class="fail">WCAG AA: Fail</span></td>
+        <td data-label="Large text"><span class="pass">WCAG AA: Pass</span></td>
+        <td data-label="Objects and UI components"><span class="pass">WCAG AA: Pass</span></td>
+      </tr>
+      <tr>
+        <td data-label="Background color"><code>black</code> (<code>#000</code>)</td>
+        <td data-label="Contrast ratio"><span class="pass">4.63</span></td>
+        <td data-label="Normal text"><span class="pass">WCAG AA: Pass</span></td>
+        <td data-label="Large text"><span class="pass">WCAG AA: Pass</span></td>
+        <td data-label="Objects and UI components"><span class="pass">WCAG AA: Pass</span></td>
+      </tr>
+    </tbody>
+  </table>
+</rh-table>
 
-If color is the only way to distinguish between links and surrounding text (e.g., if link underlines are removed), the contrast ratio between the link and surrounding text must be at least 3:1.
 
-- Non-color cues must be used to signify when the link receives hover or focus (e.g., an underline)
+### Links
+
+Inline links should always be underlined in all states. The contrast ratio between a link and surrounding text must be at least 3:1.
```

Comment:
The underlining and contrast are kind of an either/or thing. Since we're mandating underlines, we can get rid of the contrast text, to avoid confusion. I'd align this verbiage to match what was recently added to [our accessibility section](https://ux.redhat.com/accessibility/design/#links). (That page _does_ get in the weeds a bit on exceptions and contrast.)

Maybe something like this? "To avoid relying on color contrast, text links should be underlined to differentiate them from their surrounding text. Links isolated within visually distinct sections (e.g., navigation menus) or with additional visual cues (e.g., CTA arrows) do not require underlines, though you may add them at your discretion."